gradle/forge.gradle

```diff
@@ -31,6 +31,11 @@ dependencies {
         exclude group: 'org.cyclops.cyclopscore', module: 'CyclopsCore'
     }
 
+    compile "com.google.re2j:re2j:1.1"
+
+    shadow("com.google.re2j:re2j:1.1") {
+    }
```

Comment:
Is this the right way to do shading? It seems to work.

Reply:
`shadow "com.google.re2j:re2j:1.1"` should work as well. Can you try if this works? Looks a bit cleaner :-)